serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -1,13 +1,13 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-channels"]
-= Using channels
+= Using channels with Knative Eventing
```

Comment:
Should this title change also be reflected in the `topic_map.yml`?

Reply:
Decided to leave as was, looking back idk why I changed this :joy: